dtm0/fop.h

```diff
@@ -37,12 +50,52 @@ extern const struct m0_rpc_item_ops dtm0_req_fop_rpc_item_ops;
 int m0_dtm0_fop_init(void);
 void m0_dtm0_fop_fini(void);
 
+/**
+ * DTM0S operation flags.
+ */
+enum m0_dtm0s_op_flags {
+	/**
+	 * Define any operation flags
+	 */
+	DOF_NONE = 0,
+	/**
+	 * Delay reply until transaction is persisted.
+	 */
+	DOF_SYNC_WAIT = 1 << 0,
+} M0_XCA_ENUM;
+
+/**
+* DTM0S operation codes.
+*/
+
+enum m0_dtm0s_opcode{
+	DT_REQ,
+	DT_REDO,
+	DT_NR,
+} M0_XCA_ENUM;
+
+enum m0_dtm0s_msg {
+	DMT_EXECUTE_DTX,
+	DMT_EXECUTED,
+	DMT_PERSISTENT,
+	DMT_NR,
+} M0_XCA_ENUM;
```

Comment:
what about to merge all these into a single enum?
DMT_EXECUTE,
DMT_EXECUTED,
DTM_PERSISTENT,
DMT_REDO

Reply:
Done.